cg/cli/add.py

```diff
@@ -206,10 +206,10 @@ def family(
         LOG.error(f"{customer_id}: customer not found")
         raise click.Abort
 
-    for panel_id in panels:
-        panel_obj: models.Panel = status_db.panel(panel_id)
+    for panel in panels:
```

Comment:
I would use `panel_id` for the strings and `panel` for the panel object. At least I have been handling it like this. So my suggestion is to replace:
`panels` -> `panel_ids`
`panel` -> `panel_id` 
`panel_obj` -> `panel`

Reply:
It is not a panel_id confusingly enough, it is a panel abbreviation - so I will change it to that